src/Application.vala

```diff
@@ -44,7 +44,7 @@ public class PantheonTerminal.TerminalApp : Gtk.Application {
         Intl.setlocale (LocaleCategory.ALL, "");
     }
 
-    public TerminalApp () {
+    public Application () {
         Granite.Services.Logger.initialize ("PantheonTerminal");
```

Comment:
Should this be "Terminal" now? It looks like this arg is silently discarded anyway: https://github.com/elementary/granite/blob/07ff49fabb2d3617d2717b751ae5eaded60add0f/lib/Services/Logger.vala#L95